src/Executor/DryRunORMExecutor.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Common\DataFixtures\Executor;
+
+/**
+ * Class responsible for executing data fixtures in dry run mode.
```

Comment:
Let's not paraphrase the class' name and do some explaining instead

```suggestion
 * This executor allows to execute (and indirectly, print) SQL statements without
 * actually committing them to the database
```

Reply:
Done